kratos/tests/cpp_tests/geometries/test_quadrature_point.cpp

```diff
@@ -0,0 +1,107 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:     BSD License
+//           Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher
```

Comment:
:v:

Reply:
Man, who reads the Authors of a file? :P